.github/workflows/e2e-arm64-template.yaml

```diff
@@ -28,6 +28,9 @@ jobs:
       - uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
         with:
           ref: "${{ inputs.etcdBranch }}"
+          # Workflows that run 'release e2e' tests need to avoid github default shallow clone
+          # Setting fetch-depth 0 clones full project so all tags are present
+          fetch-depth: 0
```

Comment:
Don't like implicit dependency of `PASSES='build release e2e' COVER='false' ./test.sh` on local repo state. I think that `release` script should check the available releases from remote.

Reply:
Yeah fair point. I've switched to an approach based on `git ls-remote --tags`.  Please let me know what you think 🙂

Reply:
Here is the actions log for e2e showing the new approach in action:https://github.com/etcd-io/etcd/actions/runs/5585011211/jobs/10207184417?pr=16230#step:5:632